drivers/net/macb.c

```diff
@@ -920,6 +968,14 @@ static int _macb_init(struct udevice *dev, const char *name)
 		macb_writel(macb, TBQPH, upper_32_bits(macb->tx_ring_dma));
 	}
 
+	/*
+	 * Call axi bus init only for BCM2712 board which has macb
+	 * connected over PCIe.
+	 */
+	if ((IS_ENABLED(CONFIG_BCM2712)) &&
+			(gd_board_type() == BCM2712_RPI_5_B_NEW))
+		gem_init_axi(macb);
```

Comment:
Pls, solve it using mach_data.

Reply:
search for "mach_data" in entire u-boot project gives only one entry in structure vsc9953_ana_ana_tables
Maybe there is some typo in the name?